packages/cashier/src/pages/withdrawal/__tests__/withdrawal.spec.tsx

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable react/display-name */
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Just for my understanding, why we are disabling these?

Reply:
@niloo-fs wanted to avoid eslint errors, but as i can see in @farzin-fs's comment below you already added eslint ignore for test.tsx files. Will remove all this disables ...

Reply:
Thank u @george-usynin-binary